appendix-basic65.tex

```diff
@@ -3457,9 +3463,19 @@ \subsection{EDMA}
                  target [{\bf,} sub{\bf,} mod]
 \item [Usage:]
    {\bf EDMA} ("Extended Direct Memory Access") is the fastest method
-   to manipulate memory areas using the DMA controller.
+   to manipulate memory areas using the DMA controller. Please refer to
+\ifdefined\printmanual
+the {\bf MEGA65 Book}
+\else
+\bookvref{cha:dmagic}
+\fi
+   for the powerful possibilities with EDMA.
 
    {\bf command} 0: copy, 1: mix, 2: swap, 3: fill.
```

Comment:
Wait a minute! Wasn't there a comment in Discord this is getting *way* above a BASIC book beginner's brain capacity? When we first discussed that **one** direction bit should go into the manuals it was always said experts could follow the reference to the EDMA chapter. Tables like this and variants because of different DMA modes (it is not that simple, you know) we all find following the reference.

I'm not changing this unless you insist. Better ask LGB and Paul first.

{{Memo to myself: Never again file or follow issues without screenshots of Discord threads.}} 🙄

Reply:
Hehe, no worries about always linking discord threads. Sometimes I do link the threads in to help jog my memory on past convos, so if we find the old threads and we can post-humously link it in here, no problem :)

As for my thoughts above, they were a suggestion, open to be talked over, I'm not enforcing those thoughts with a hammer ;) Think I got the impression from LGB on a recent discord comment that he was keen to see more hidden bits documented, that's why the idea to add the table came to my mind. Though perhaps others in the team felt otherwise in past convos that you witnessed, fair enough.

Agree that this table is a little problematic as is, since the bit-field meanings vary depending on whether your dma if F018A or F018B.

I had shared this annotated snapshot of LGB's diagram on discord, where we see that for F018A, these bits will not impact direction, they will only behave that way fro F018B.

So given that, it might cast a shadow over whether we should even describe the existence of a 'direction' bit in this section here. Or perhaps we should preface it with a pre-caution that it is only a 'direction' bit if your DMA is F018B.

Reply:
I'm thinking, for now, let's leave it as you have it, and we can ponder the F018A/B variations at a later time.